opte/Cargo.toml

```diff
@@ -51,5 +43,10 @@ default-features = false
 #
 features = ["alloc", "medium-ethernet", "proto-ipv4", "proto-ipv6", "proto-dhcpv4", "socket", "socket-raw"]
 
+[build-dependencies]
+version_check = "0.9.4"
+
 [dev-dependencies]
```

Comment:
So it looks like usdt is always enabled now for test? That's good, I should have done that to begin with, as I can see from you other modifications I let some of the probe sites rot.

Reply:
Yup, tested on both illumos and macOS. Seems to work:

```
$ sudo dtrace -Z -f "*opte*" | rustfilt
dtrace: system integrity protection is on, some features will not be available

dtrace: description '*opte*' matched 0 probes
CPU     ID                    FUNCTION:NAME
  6  35374 opte::engine::port::Port<N>::port_process_entry_probe:port-process-entry 
  6  35372 opte::engine::layer::Layer::layer_process_entry_probe:layer-process-entry 
  6  35377 opte::engine::layer::RuleTable::rule_match_probe:rule-match 
  4  35374 opte::engine::port::Port<N>::port_process_entry_probe:port-process-entry 
[...snip...]
```